CHANGELOG.md

```diff
@@ -13,12 +13,17 @@ Go to [legend](#legend---types-of-changes) for further information about the typ
 
 ### Added
 
+- Files to fulfill the community standards
+	- bug_report.md and feature_request.md
+	- CODE_OF_CONDUCT.md file
+	- PULL_REQUEST_TEMPLATE.md file
 - _WD_JsonCookie() (mLipok)
 
 ### Changed
 
 - README.md restruturing
```

Comment:
> My review was focused on line 24, and this is something you have to get used to on github.
btw. I only noticed it myself last week.

If you click on the row number of the target row before clicking the plus sign, then it should highlight the correct row for the reviewer. This is a test of that functionality.

Reply:
Hmmm... I guess it doesn't work the way I thought. ☹️